### PR TITLE
issue-template: move hashtags to issue body

### PR DESCRIPTION
**Pull request by sterchelen**
ref #35

### CODE REVIEW

**Comment by pabluk** [comment]
Thanks for this PR, n order to help you with this refacto I added a test on this commit https://github.com/sre-paris/reliability.re/commit/c5ffd8afb07071ddc8c57d10196d936355ae072c, it will allow you to have a way of test your changes and to see the results

File: .github/ISSUE_TEMPLATE/new-content.md [hunk 1]
```diff
@@ -4,11 +4,9 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-hashtags:
-  - hash1
-  - hash2
 ---
 
 url: https://example.com/
+hashtags: #hash1,#hash2
```
Comment (pabluk) [inline .github/ISSUE_TEMPLATE/new-content.md:10]: Regarding this new line, how do you think that we could manage the case when someone don't want to use any hashtag? can they just remove this line or do they need to leave it empty?

Comment (sterchelen) [reply .github/ISSUE_TEMPLATE/new-content.md:10]: Yep with this https://github.com/sre-paris/reliability.re/pull/37/files#diff-dc96b1335dec01e5a2533179be0dc7a142292f4c67a06390dc33a852259dfb30R66

Comment (sterchelen) [reply .github/ISSUE_TEMPLATE/new-content.md:10]: Yes, thanks to this 4c1d370e0ded8a0094ab41a2ba51ab9c065f92af

Would be easier to maintain by iterating over the content, and based on characters before the first `:` use the strategy pattern; `url:` then call the right strategy,...

Comment (sterchelen) [reply .github/ISSUE_TEMPLATE/new-content.md:10]: On twitter I added a test to check if `hashtags` is present.

**Review by pabluk — APPROVED** [review]
Looks great! Now hashtags for Twitter will be added automatically and tests are passing with no issues! :rocket:
Thanks again @sterchelen!

**Pull request merged**